x-pack/plugins/rollup/index.js

```diff
@@ -52,6 +54,12 @@ export function rollup(kibana) {
       registerSearchRoute(server);
       registerJobsRoute(server);
       registerRollupUsageCollector(server);
+      if (
```

Comment:
@bmcconaghy 
I think it will be better if you move this logic to e.g. registerIndexManagementDataEnricher function. 

Please see we deal with it: https://github.com/elastic/kibana/pull/28762/files#diff-02dfe83a6810d9d61268f6c6dd28ad80 
Feel free to add your comments

FYI: @jen-huang, @timroes